vocode/streaming/transcriber/whisper_cpp_transcriber.py

```diff
@@ -60,16 +64,18 @@ def create_new_buffer(self):
         wav.setframerate(self.transcriber_config.sampling_rate)
         return wav, buffer
 
-    async def run(self):
+    async def _run_loop(self):
         in_memory_wav, audio_buffer = self.create_new_buffer()
         message_buffer = ""
         while not self._ended:
-            chunk = await self.audio_queue.get()
+            chunk = await self.input_queue.get()
             in_memory_wav.writeframes(chunk)
             if audio_buffer.tell() >= self.buffer_size * 2:
                 audio_buffer.seek(0)
                 audio_segment = AudioSegment.from_wav(audio_buffer)
-                message, confidence = transcribe(
+                message, confidence = await asyncio.get_event_loop().run_in_executor(
```

Comment:
This transcriber seems like a good candidate for the ThreadAsyncWorker. This seems very inneficient to call this every 20ms (in Twilio's case) just to send a tiny buffer of audio. 

(Btw I didn't realize Whisper had non-batch / streaming interface. I thought it could only process file vs stream of bytes)

Reply:
Whisper doesn't have a non-batch interface, since it's so fast locally we're just running the forward pass with the buffer so far and re-initializing the buffer on a sentence end (e.g. `.`)

good call on the ThreadAsyncWorker, switching it